rdkit-postgresql95/meta.yaml

```diff
@@ -4,23 +4,27 @@ package:
 
 source:
   git_url: https://github.com/rdkit/rdkit.git
-  git_tag: Release_2016_09_2
+  git_rev: Release_2017_03
   patches:
-    - Makefile.patch
+    - cmakelists.txt.patch
     - adapter.cpp.patch
 
 build:
   number: 1
 
 requirements:
   build:
-    - python
-    - rdkit >=2016.09
-    - postgresql95
+    - cmake
+    - msys2-conda-epoch >=20160418 [win]
+    - m2-msys2-runtime [win]
+    - m2-libiconv [win]
+    - m2-libintl [win]
+    - m2-diffutils [win]
+    - m2-patch [win]
+    - postgresql >=9.5
```

Comment:
but this way the cartridge will require the 'postgresql' package to be installed at runtime, making it in practice incompatible with the installation of 'postgresql95' in the same environment. 

maybe we could drop the '95' suffix from the package name produced by the 'postgresql95' recipe, and this way allow some flexibility, at the (probably reasonable?) cost or requiring the users to pin the installed database version to avoid accidental upgrades across different versions of postgres. 

(removing the same suffix from the  'rdkit-postgresql*' package name is not possible with conda-build 2.x, because the package filename wouldn't reflect the different dependency).